### PR TITLE
Update mac build instructions for 2.6

### PR DESCRIPTION
**Pull request by devvmh**
~~This is still in progress.~~ Testing todo list:

- [x] Able to run `make macos` with no breaking errors
- [x] Check through build logs for errors
- [x] Open dist/Mnemosyne.app; use UI for a bit; open preferences
- [x] Open browse cards via menu and Cmd+B
- [x] look at graphs
- [x] enable/disable a plugin
- [x] add cards
- [x] remove cards
- [x] edit + preview cards
- [x] Import cards
- [x] `brew uninstall mplayer` then test audio cards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,19 +214,21 @@ brew install qt@5.7 mplayer
 pip3 install virtualenv
 virtualenv --python=python3 venv
 source venv/bin/activate
-pip3 install webob tornado matplotlib numpy sip pillow cheroot pyqt5 pyinstaller
+pip3 install webob tornado matplotlib numpy sip pillow cheroot decorator pyqt5==5.7.1 pyinstaller
```
Comment (three-comrades) [inline README.md]: I have no idea about MacOS, but out of curiosity: why is it required to pin the pyqt5 version here?

Comment (devvmh) [reply README.md]: Packaging with pyqt5 is a total crapshoot. I pin the version of qt5 in homebrew and pyqt5 here to 5.7 to minimize churn. 

If i get a good install tonight ill consider upgrading to 5.8 or later

**Comment by devvmh** [comment]
build log errors:

5832 ERROR: Hidden import 'mnemosyne.libmnemosyne.plugins.cramming_plugin' not found
6005 ERROR: Hidden import 'mnemosyne.libmnemosyne.ui_components.headless_review_widget' not found
6011 ERROR: Hidden import 'mnemosyne.libmnemosyne.upgrades.upgrade3' not found
14797 ERROR: Hidden import 'cherrypy.wsgiserver.wsgiserver3' not found
25534 ERROR: Can not find path /opt/local/lib/mysql55/mysql/libmysqlclient.18.dylib (needed by /Users/devin/git/mnemosyne/venv/lib/python3.6/site-packages/PyQt5/Qt/plugins/sqldrivers/libqsqlmysql.dylib)

None of these seem to result in actual bugs, although I don't know how to test the data uploads to the server.

I'm calling this mergeable. I've got a built package locally which I can rebuild if more is added on the day of release

File: README.md (hunk 1 above)
Comment (devvmh) [reply README.md]: I tried building with `brew install qt@5.9` and `pip3 install pyqt5==5.9 but I get an error 

    Illegal instruction: 4

I'll try again for Mnemosyne 2.7 haha

**Comment by pbienst** [comment]
The hidden import things should be harmless, they refer to files which I removed, but I forgot to update the *.spec file. Will do so later.

Feel free to send me a link to the binary from the current head, then I'll add it to the same location as the other RC1 test files.

**Pull request merged**